Microsoft.DotNet.Interactive.PowerShell/Commands/OutDisplayCommand.cs

```diff
@@ -26,8 +26,22 @@ public sealed class OutDisplayCommand : PSCmdlet
         /// <summary>
         /// The MimeType to use.
         /// </summary>
-        [Parameter(Position = 1)]
-        public string MimeType { get; set; }
+        [Parameter(Position = 1, ParameterSetName = "MimeType")]
+        [ValidateSet(
+            "application/javascript",
```

Comment:
@brettfo I think we should handle both `text/javascript` and `application/javascript` mimetypes in your PR #251